ironmon_tracker/screens/TrackerScreen.lua

```diff
@@ -492,9 +495,11 @@ function TrackerScreen.drawScreen()
 	end
 
 	-- Add in Pokedex information about the Pokemon
-	local pokedexInfo = Utils.inlineIf(viewedPokemon.pokemonID ~= 0, PokemonData.Pokemon[viewedPokemon.pokemonID], PokemonData.BlankPokemon)
-	for key, value in pairs(pokedexInfo) do
-		viewedPokemon[key] = value
+	if PokemonData.isValid(viewedPokemon.pokemonID) then
+		local pokedexInfo = PokemonData.Pokemon[viewedPokemon.pokemonID]
```

Comment:
Removes copying BlankPokemon data in when the pokemonID is 0. This is addressed by getDefaultPokemon now containing that data, and if viewedPokemon.ID == 0 we were already loading in the Default Pokemon

Reply:
I like this solution for the time being. At some point in the future we should merge the default pokemon values assigned in the Tracker class and the PokemonData class to use the exact same values (don't repeat typing in duplicate values).